agent.go

```diff
@@ -1060,6 +1062,12 @@ func addMounts(config *configs.Config, fsmaps []hyper.Fsmap) error {
 func newContainerCb(pod *pod, data []byte) error {
 	var payload hyper.NewContainer
 
+	// re-scan PCI bus
+	// looking for hidden devices
+	if err := ioutil.WriteFile(pciBusRescanFile, []byte("1"), pciBusMode); err != nil {
```

Comment:
@devimc In the other PR you are reusing the pci slots. Not sure if a slot will ever be reused (block device unplug followed by a plug in the real world with virtcontainers). However if that does happen, what happens to the mount points?

Also how do we know when the rescan is complete?

Reply:
@mcastelino pci slots can be reused once it's free

> what happens to the mount points?

There is still missing that part, in qemu <= 2.9 Q35 PCI devices must be removed manually, I guess we have to implement an extra command in the agent to clean up unplugged devices

> Also how do we know when the rescan is complete?
it is a blocking IO operation, process will not continue until all PCI buses are re-scanned

Reply:
@devimc will the pci slot reuse happen in the real world. Do we support removing a container from a POD?

Reply:
not sure if we can remove a container from a POD, but we can unplug devices from bridges.
we could reuse that slot to hot plug another device if we need it since we have a limitation in the number of devices per bridge.

Reply:
@devimc so when will we ever unplug a device? I cannot think of a case. Which means we are ok for now and need no special logic or unmount/remount.